source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -35,16 +35,18 @@ namespace
     static const RtTokenSet attrMetadata        = { RtToken("name"), RtToken("type"), RtToken("value"), RtToken("nodename"), RtToken("output"), RtToken("channels") };
     static const RtTokenSet inputMetadata       = { RtToken("name"), RtToken("type"), RtToken("value"), RtToken("nodename"), RtToken("output"), RtToken("channels"), 
                                                     RtToken("nodegraph"), RtToken("interfacename") };
-    static const RtTokenSet nodeMetadata        = { RtToken("name"), RtToken("type"), RtToken("node") };
-    static const RtTokenSet nodegraphMetadata   = { RtToken("name") };
-    static const RtTokenSet genericMetadata     = { RtToken("name"), RtToken("kind") };
+    static const RtTokenSet nodeMetadata        = { RtToken("name"), RtToken("type"), RtToken("node"), RtToken("xpos"), RtToken("ypos") };
+    static const RtTokenSet nodegraphMetadata   = { RtToken("name"), RtToken("xpos"), RtToken("ypos") };
+    static const RtTokenSet genericMetadata     = { RtToken("name"), RtToken("kind"), RtToken("xpos"), RtToken("ypos") };
```

Comment:
Need to declare that xpos/ypos is now handled explicitly and should be ignored when reading general metadata.